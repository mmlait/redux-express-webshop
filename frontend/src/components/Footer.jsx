import React from 'react';
import styled from 'styled-components';
import colors from '../colors';

const AppFooter = styled.footer`
  color: ${colors.secondary};
  text-align: center;
  position: absolute;
  bottom: 0;
  width: 100%;
  height: 160px;
  display: flex;
  align-items: center;
  justify-content: center;
`

const Footer = () => {
  return (
    <AppFooter>
      <span>&copy;2019 Miia Laitinen</span>
    </AppFooter>
  );
}

export default Footer;