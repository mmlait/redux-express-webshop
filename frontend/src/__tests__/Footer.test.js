import React from 'react';
import Footer from '../components/Footer';
import { create } from 'react-test-renderer'

describe('My first snapshot test', () => {
    it('should render Footer component', () => {
    let tree = create(<Footer />)
    expect(tree.toJSON()).toMatchSnapshot();
  })
})