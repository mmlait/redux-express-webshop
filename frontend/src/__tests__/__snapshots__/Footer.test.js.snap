// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render 1`] = `
<footer
  className="sc-bdVaJa hlRHXh"
>
  <span>
    Â©2019 Miia Laitinen
  </span>
</footer>
`;
